pyiron/vasp/base.py

```diff
@@ -724,6 +748,25 @@ def write_magmoms(self):
         else:
             s.logger.debug("No magnetic moments")
 
+    def set_eddrmm(self, status="not_converged"):
```

Comment:
I wonder if it's better to rename this to `set_eddrmm_handling` and `get_eddrmm_handling`? `set_eddrmm` implies you are setting a parameter.

Reply:
sounds good!